test/mangle.coffee

```diff
@@ -161,3 +161,20 @@ describe 'mangle:', ->
                 distinguishFunctionExpressionScope: yes
             expect(result.body[0].expression.id.name).to.equal 'a'
             expect(result.body[0].expression.body.body[0].declarations[0].id.name).to.equal 'a'
+
+    describe '`cb` option:', ->
```

Comment:
Done. I've also fixed indentation (I was using two spaces instead of 4) and added a new test to cover the scenario where no `shouldRename` callback is provided.